### PR TITLE
Include OpenEXR and LEMON

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Any chance this could be merged before migrating to python 3.8?

**Comment by stuarteberg** [comment]
LGTM!  Would you like to help maintain this, @hmaarrfk?  If so, add yourself as a maintainer.  Otherwise, I'll just merge as-is.

**Comment by hmaarrfk** [comment]
thx for the fast turnaround. Added myself in an other PR.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -74,17 +74,20 @@ requirements:
...
     - zlib
-    
+
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - jpeg
     - libtiff
     - libpng
+    - lemon
```
Comment (jakirkham) [inline recipe/meta.yaml:90]: FWIW `lemon` only has a static library. So we can probably drop it from `run`.

Comment (hmaarrfk) [reply recipe/meta.yaml:90]: ah ok. thx.

Comment (stuarteberg) [reply recipe/meta.yaml:90]: And I think we need to add `openexr` to the `run` requirements.  Looks like it was accidentally dropped when `lemon` was added.

Geez, I failed as a reviewer here... sorry guys.

Comment (hmaarrfk) [reply recipe/meta.yaml:90]: No, that one has a run exports
https://github.com/conda-forge/openexr-feedstock/blob/master/recipe/meta.yaml#L18

Comment (stuarteberg) [reply recipe/meta.yaml:90]: oh ok.  Sorry for the noise, then.

Comment (hmaarrfk) [reply recipe/meta.yaml:90]: not an issue

Comment (jakirkham) [reply recipe/meta.yaml:90]: It's all good 🙂

**Pull request merged**